platform/javascript/os_javascript.cpp

```diff
@@ -702,7 +727,11 @@ bool OS_JavaScript::can_draw() const {
 
 void OS_JavaScript::set_cursor_shape(CursorShape p_shape) {
 
-	//javascript really really really has no mouse.. how amazing..
```

Comment:
Yeah that's looking silly now =p
Those comments are actually remnants from OS_Android though, OS_JavaScript was copy/pasted from there